meta/src/store/state_machine.rs

```diff
@@ -647,6 +647,12 @@ impl StateMachine {
     }
 
     fn process_add_date_node(&self, cluster: &str, node: &NodeInfo) -> CommandResp {
+        for (_, value) in children_data::<NodeInfo>(&KeyPath::data_nodes(cluster), self.db.clone()).iter() {
+            // Check whether the ip of the same cluster has been added
+            if value.id == node.id && value.http_addr == node.http_addr {
```

Comment:
The relationship between node id and httpaddr is one-to-one, the logic of the if statement should be or, and tcpaddr should also be added.

Reply:
Thanks, I will fix these issues.